nursereg/tasks.py

```diff
@@ -255,6 +259,32 @@ def create_subscription(contact, sender=None):
             exc_info=True)
 
 
+def transfer_subscription(contact, rm_active_nr_subs):
+    for sub in rm_active_nr_subs:
+        # activate the same subscriptions on the new msisdn
+        subscription = Subscription(
+            contact_key=contact["key"],
+            to_addr=contact["msisdn"],
+            user_account=contact["user_account"],
+            lang="en",
+            message_set=sub.message_set,
+            schedule=sub.schedule,
+            next_sequence_number=sub.next_sequence_number)
+        subscription.save()
+
+        # deactivate active subscriptions for rmsisdn
+        sub.active = False
+        sub.save()
+
+    # TODO #123: Clear extras for old contact for external change requests
+
+    # return the last created subscription
+    cm_active_nr_subs = Subscription.objects.filter(
```

Comment:
This is only going to work as long as there's only one nurse related messageset. I don't think we need to address this now, but please note so in a comment